src/plugins/search/search.js

```diff
@@ -208,10 +208,10 @@ export function init(config, vm) {
 
     if (Array.isArray(config.pathNamespaces)) {
       namespaceSuffix =
-        config.pathNamespaces.find(prefix => path.startsWith(prefix)) ||
+        config.pathNamespaces.find(prefix => path && path.startsWith(prefix)) ||
```

Comment:
What about checking for the `paths.length` here: https://github.com/docsifyjs/docsify/pull/1358/files#diff-4ce3780eb5b2c8eb805df4e0c950cd7dR206 ?

If I understood correctly, the problem occurs only when there is nothing to index, i.e., `paths == []`. If so, adding a `paths.length` would be more descriptive.

Reply:
Are there any cases where `path` will be an array? I think there are non, cause if so I think checking length makes sense.
Also, arrays don't contain the method `startsWith`. 

Let me know if I am missing something here.

Reply:
I meant `paths`, not `path`. See line 201 (and we can add an outer `if (paths.length) { ... }` clause before line 206).

Reply:
done.

@anikethsaha An array if _sidebar exists, otherwise empty.

Reply:
COol